lib/tic_tac_toe/web/game_presenter.ex

```diff
@@ -0,0 +1,11 @@
+defmodule TicTacToe.Web.GamePresenter do
+
+  def render_game(game) do
+    board_width = Board.width(game.board)
+    rows = game.board.cells |> Enum.with_index |> Enum.chunk(board_width)
```

Comment:
I'm wondering if this logic to extract the rows should belong on the board. Maybe a violation of Tell, Don't Ask? Also, `game.board.cells` may almost be a violation of the Law of Demeter too.